flixel/tile/FlxTilemap.hx

```diff
@@ -128,9 +128,18 @@ class FlxTilemap extends FlxBaseTilemap<FlxTile>
 	private var _scaledTileHeight:Float = 0;
 	
 	#if FLX_DEBUG
-	private var _debugTileNotSolid:BitmapData;
-	private var _debugTilePartial:BitmapData;
-	private var _debugTileSolid:BitmapData;
+	private var updateDebugTileBoundingBoxesInit = false;
```

Comment:
i'd rather remove this var and check if `_tileWidth` and `_tileHeight` are initialized (have values more than 0).

so we could have something like:
```
override private function set_debugColorScheme(value:DebugColorScheme):DebugColorScheme
	{
		this.debugColorScheme = value;
		
		if (_tileWidth > 0 && _tileHeight > 0)
		{	
			updateDebugTileBoundingBoxes();
		}
		
		return value;
	}
```